tutoraspects/asset_command_helpers.py

```diff
@@ -225,15 +225,35 @@ class DatabaseAsset(Asset):
 }
 
 
+def _check_file_exists(out_path, filename):
+    """
+    Check if file exists and return loaded yaml
+    """
+    if os.path.exists(os.path.join(out_path, filename)):
+        with open(os.path.join(out_path, filename), encoding="utf-8") as stream:
+            return yaml.safe_load(stream)
+    return None
+
+
 def validate_asset_file(asset_path, content, echo):
     """
     Check various aspects of the asset file based on its type.
 
     Returns the destination path for the file to import to.
+    Append last 6 characters of uuid for charts
     """
     orig_filename = os.path.basename(asset_path)
-    out_filename = re.sub(r"(_\d*)\.yaml", ".yaml", orig_filename)
-    content[FILE_NAME_ATTRIBUTE] = out_filename
+    out_filename_trimmed = re.sub(r"(_\d*)\.yaml", ".yaml", orig_filename)
```

Comment:
Nit: can you move this into the `else` below so we're not having to do this unnecessarily?